<launch>
   <node name="msf_position_sensor" pkg="msf_updates" type="position_sensor" clear_params="true" output="screen">
            <remap from="msf_core/hl_state_input" to="/fcu/ekf_state_out" />
            <remap from="msf_core/correction" to="/fcu/ekf_state_in"/>

	    <remap from="msf_core/imu_state_input" to="/auk/fcu/imu" />
	    <remap from="msf_updates/position_input" to="/leica/position" />
	    <!--remap from="msf_updates/navsatfix_input" to="/auk/fcu/gps" /-->
	    	

            <!--remap from="msf_updates/transform_input" to="vicon/auk/auk" /--> 

            <rosparam file="$(find msf_updates)/position_sensor_fix.yaml"/>
    </node>
    <node pkg="rosservice" type="rosservice" name="initialize" args="call --wait /msf_position_filter/position_sensor/initialize_msf_scale 1"/>
</launch>

